api/src/main/java/org/openmrs/module/locationbasedaccess/LocationBasedAccessActivator.java

```diff
@@ -46,6 +51,14 @@ public void willStart() {
      * @see ModuleActivator#started()
      */
     public void started() {
+        if (personService.getPersonAttributeTypeByName("accessControlLocation") == null) {
+            PersonAttributeType personAttributeType = new PersonAttributeType();
+            personAttributeType.setUuid(ACCESS_LOCATION_UUID);
```

Comment:
What does access location mean?

Reply:
Is it better to rename as ACCESS_CONTROL_LOCATION_UUID?

Reply:
It depends on the use case. What is this uuid to be used for?

Reply:
This will be used to create the PersonAttribute with that given UUID. So I can simply define the fragement with this UUID as follows,  https://gist.github.com/suthagar23/f0b92381aec5d0ad6d70e4eb40eb308d#file-appdefinition-L88

Reply:
That i understand. But what exactly does this uuid stand for?

Reply:
What happens if some one wants to use a different uuid for the person attribute type?

Reply:
Not a big changes here, he need to get that person Attribute uuid from the system and replace that uuid in the app definition.
If we use this predefined uuid, then no need to change that app definition. Just copy and paste it enough :-)

Reply:
Doesn't this code overwrite it at every module startup?

Reply:
Nope @dkayiwa, It will only create if the person Attribute doesn't exist in the system.